config/sbnd/sbnd_full_chain_240918.cfg

```
@@ -539,7 +539,7 @@ post:
   calo_ke:
     run_mode: both
     scaling: 1.
-    shower_fudge: 1/0.83
+    shower_fudge: 1/0.77 # previous:  1/0.83
```

```suggestion
    shower_fudge: 1/0.77
```
Same here, we don't need the previous value since you can find it in the commit history